nginx-proxy-manager/umbrel-app.yml

```
@@ -0,0 +1,47 @@
+manifestVersion: 1
+id: nginx-proxy-manager
+name: Nginx Proxy Manager
+tagline: Expose your services easily and securely
+icon: https://nginxproxymanager.com/logo.svg
+category: networking
+version: "2.11.3"
+port: 81
+description: >-
+  Expose your services easily and securely
+  
+  
+  Features:
+  
+      - Beautiful and Secure Admin Interface based on Tabler: https://tabler.github.io/
+      
+      - Easily create forwarding domains, redirections, streams and 404 hosts without knowing anything about Nginx
+      
+      - Free SSL using Let's Encrypt or provide your own custom SSL certificates
+      
+      - Access Lists and basic HTTP Authentication for your hosts
+      
+      - Advanced Nginx configuration available for super users
+      
+      - User management, permissions and audit log
+      
+  
+  ⚠️ Please take care while exposing apps to the internet.
+  
+  ⚠️ Note: Since Port 80 is used my Umbrel itself, it will not be handled by Nginx Proxy Manager. Hence do not to forward Port 80 from your router. Only forward Port 443. ⚠️
+  
+developer: Jamie Curnow (jc21.com)
```

We can remove the website link here
```suggestion
developer: Jamie Curnow
```